commands/support/suggest.js

```diff
@@ -34,11 +34,12 @@ class SuggestCommand extends Commando.Command
             .addField('Sugestion', reason)
             .setFooter("Click the green check to like the idea or the red x if you don't like the idea!")
         let logchannel = message.guild.channels.find('name', 'suggestions'); 
-        logchannel.send(Suggestmsg).then(embedMessage => {
+        logchannel.send(Suggestmsg).then(embedMessage => {:
```

Comment:
Placing the : on line 37 will break the code resulting in a crash or the reactions not working.

Reply:
`logchannel.send(Suggestmsg).then(embedMessage => {`